### PR TITLE
Upgrade various QT components

### PR DESCRIPTION
**Pull request by imduffy15**
Upgrade to QT 5.14.1
Upgrade PyQT to 5.13.2 (They've dropped GPL in the filename, I don't see any mention of license changes)
Upgrade PyQTWebengine to 5.13.2 (They've dropped GPL in the filename, I don't see any mention of license changes)
Upgrade QtWebkit to 5.212.0 Alpha 4

### CODE REVIEW

File: dockerfiles/splash/download-qt-installer.sh [hunk 1]
```diff
@@ -2,6 +2,6 @@
 
 # XXX: if qt version is changed, Dockerfile should be updated,
 # as well as qt-installer-noninteractive.qs script.
-URL="http://download.qt.io/official_releases/qt/5.13/5.13.1/qt-opensource-linux-x64-5.13.1.run"
+URL="http://download.qt.io/official_releases/qt/5.14/5.14.1/qt-opensource-linux-x64-5.14.1.run"
```
Comment (kmike) [inline dockerfiles/splash/download-qt-installer.sh:5]: Why not 5.14.2?

**Comment by kmike** [comment]
That awesome @imduffy15! 

Do you think you can pull @whalebot-helmsman's changes from #1037 here as well? Or would it be easier to do this the other way around? @imduffy15 @whalebot-helmsman what do you think?

**Comment by whalebot-helmsman** [comment]
@kmike let me combine them into a single one

File: dockerfiles/splash/download-qt-installer.sh (hunk 1 above)
Comment (imduffy15) [reply dockerfiles/splash/download-qt-installer.sh:5]: https://github.com/qtwebkit/qtwebkit/releases/tag/qtwebkit-5.212.0-alpha4 said to use 5.14.1, I'm sure the patch version wouldn't make too much difference but wanted to follow the instructions.

**Comment by imduffy15** [comment]
Yeah huge plus one to merging in with @whalebot-helmsman , we're working towards a common goal in both PRs :)

File: dockerfiles/splash/download-qt-installer.sh (hunk 1 above)
Comment (kmike) [reply dockerfiles/splash/download-qt-installer.sh:5]: makes sense!

**Comment by imduffy15** [comment]
Closing in favor of https://github.com/scrapinghub/splash/pull/1037#event-3396840136

**Pull request closed without merging**